lib/newman_conway.py

```diff
@@ -1,10 +1,20 @@
 
 
-# Time complexity: ?
-# Space Complexity: ?
-def newman_conway(num):
-    """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
-    """
-    pass
+# Time complexity: ? O(n^2)
+# Space Complexity: ? O(1)
+
+def newman_conway(n):
```

Comment:
👍  The time complexity is O(n) (one loop) and the space complexity O(n) because you keep the memo `f`